packages/sdk-core/src/model/data/BacktraceStackTrace.ts

```diff
@@ -0,0 +1,12 @@
+export interface BacktraceStackFrame {
+    funcName: string;
+    line?: number;
+    column?: number;
+    sourceCode?: string;
+    library: string;
+}
+export interface BacktraceStackTrace {
+    name: string;
+    fault: boolean;
```

Comment:
What does `fault` mean? Can you add descriptions?

Reply:
api.backtrace.io